edav_info_categorical_variables_Chinese_translation.Rmd

```diff
@@ -0,0 +1,289 @@
+# Chinese Translation of Bar Chart and Cleveland Dot Plot 
+
+Changhao He
+
+Source: https://edav.info/bar.html<br/>
+        https://edav.info/cleveland.html
+
+## 1.图表：条形图
```

Comment:
Please remove the numbers after the hashtag. The hashtags will automatically create the numbered subheadings. Please refer point number 5 here:
https://jtr13.github.io/cc21/tutorial-for-pull-request-mergers.html#check-.rmd-file-contents

This is how it should look:
```suggestion
## 图表：条形图
```
Please change for all following subheadings in the file too.

Reply:
Removed numbers after hashtags.